src/openslide-vendor-dicom.c

```diff
@@ -536,6 +568,19 @@ static bool is_type(char **type, const struct allowed_types *types) {
   return false;
 }
 
+static void rgb_to_cairo(const char *rgb,
+			 uint32_t *dest,
+			 int64_t width,
+			 int64_t height)
+{
+  int64_t n_pixels = width * height;
+
+  for (int64_t i = 0; i < n_pixels; i++) {
+    dest[i] = 0xff000000 | rgb[0] << 16 | rgb[2] << 8 | rgb[2];
```

Comment:
Was this meant to be `rgb[1] << 8`?

Reply:
Ooops! Nice find.

Reply:
Thanks for the fix.  Please don't add merge commits to PR branches, though.  Something like `git fetch jcupitt add-dicom-sparse-tile-support && git reset --hard FETCH_HEAD` before adding the commit (or perhaps `git pull --rebase --interactive` if you want to preserve some existing commits) should pull new changes into your branch.

Reply:
(And in fact, it looks like the merge commit reintroduced a [line](https://github.com/openslide/openslide/compare/1f5509582ca873d3d0be905a9e9794b40eb30f56..3f0d44005104ac8d7d5220013700de38638e2a19) that had been removed.)